tests/test_data.py

```diff
@@ -901,6 +901,14 @@ def test_add_defaults(self):
         node = dnode.find_path("/yolo-system:conf/speed")
         self.assertIsInstance(node, DLeaf)
         self.assertEqual(node.value(), 4321)
+        node.unlink()
+        node.free()
+        node = dnode.find_path("/yolo-system:conf/speed")
+        self.assertIsNone(node)
+        dnode.add_defaults(only_node=True)
+        node = dnode.find_path("/yolo-system:conf/speed")
+        self.assertIsInstance(node, DLeaf)
+        self.assertEqual(node.value(), 4321)
```

Comment:
This test is not different from the only_node=False case, can you add a more specific test case?

Reply:
Done